onlineapi/src/test/java/com/codingblocks/onlineapi/api/OnlineJsonApiAuthenticatedTest.kt

```diff
@@ -47,4 +47,12 @@ class OnlineJsonApiAuthenticatedTest {
         assertNotNull(runAttempt)
         assertTrue(runAttempt!!.id == "80179")
     }
+
+    @Test
+    fun `GET wishlist`() {
```

Comment:
add a test to add a course to wishlist the get wishlist and finally remove from wishlist

Reply:
How to add that checkIfWishlisted test because it returns null if the course is not wishlisted by the user

Reply:
add a test to add a course to wishlist then get wishlist then check a particular course and finally remove from wishlist

Do I need to add these into one test only? or Seperate tests?

Reply:
API test are meant to be independent so kindly revert to the old format

Reply:
Old format means that seperate test for each API?